tore.stac_format import (

```diff
@@ -0,0 +1,76 @@
+[1mdiff --git a/geostore/check_stac_metadata/utils.py b/geostore/check_stac_metadata/utils.py[m
```

Comment:
What is this file?

Reply:
Aw crap. Not sure what created it, but I'll get rid of it.